packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -36,6 +36,7 @@ export interface SdkEmitterOptions {
   "package-name"?: string;
   "flatten-union-as-enum"?: boolean;
   "api-version"?: string;
+  "decorators-white-list"?: string[];
```

Comment:
If I understand correctly it feels like we are bleeding internal implementation into something the service team needs to understand.  I don't think we want customers to need to know what decorators to choose to include and which ones not to.  This feels like a handshake between TCGC and each emitter and the service team should not be involved?  What if the generator for a given langauge needs a decorator added and the service team doesn't add it in?

Reply:
though emitter could change it when `createSdkContext`, but i agree it should not be exposed to service. i plan to add it as an option of `createSdkContext`, is that good?

Reply:
Yeah plus one do not expose that to the user. In the same way you configure the emitter name when creating the context you should be able to configure that